test/shows.jl

```diff
@@ -0,0 +1,68 @@
+@testset "Shows" begin
+  io = IOBuffer()
+  @testset "Sequential" begin
+    t1 = Select(:x, :z)
+    t2 = ZScore()
+    t3 = Scale(low=0, high=1)
+    pipeline = t1 → t2 → t3
+
+    # compact mode
+    show(io, pipeline)
+    iostr = String(take!(io))
+    @test iostr == "Select{Tuple{Symbol, Symbol}}((:x, :z)) → ZScore() → Scale{Int64}(0, 1)"
```

Comment:
I think there is a shorter code for testing these prints: https://github.com/JuliaEarth/GeoStatsBase.jl/blob/d43cb39486a3f9d87d68276a0366f26774689de0/test/ensembles.jl#L28